modules/templates/helper.go

```diff
@@ -377,6 +377,12 @@ func NewFuncMap() []template.FuncMap {
 			html += "</span>"
 			return template.HTML(html)
 		},
+		"cond": func(condition bool, a, b interface{}) interface{} {
+			if condition {
+				return a
+			}
+			return b
+		},
```

Comment:
The designers of golang were very determined not to have the ternary operator...

```suggestion
```

Reply:
How about `cond` in Hugo? https://gohugo.io/functions/cond/

Reply:
If I had to guess this will be a throwback to pre go 1.11 when variable reassignment was not possible in go templates - however it is now possible. 

Further note:

```
Whenever you use a cond function, both variable expressions are always evaluated. This means that a usage like cond false (div 1 0) 27 will throw an error because div 1 0 will be evaluated even though the condition is false.

In other words, the cond function does not provide short-circuit evaluation and does not work like a normal ternary operator that will pass over the first expression if the condition returns false.
```

The inability to short-circuit makes `cond` substantially more error-prone and gotcha-ful.